File: Python/Chapter 1/3.py
def gcd(a, b):
    return a if b == 0 else gcd(b, a % b)

def lcm(a, b):
    return a * b / gcd(a, b) 

if __name__ == '__main__':
    print ('Lowest common multiple for 2 values')
    print ('Input a: ', end='')
    a = eval(input())
    print ('Input b: ', end='')
    b = eval(input())

    print (f'lcm({a}, {b}) = ', lcm(a, b))

File: C++/Chapter 1/3.cpp
#include <iostream>

int gcd(unsigned int a, unsigned int b){
    return b == 0? a: gcd(b, a % b); 
}

int lcm(unsigned int a, unsigned int b){
    return a * b / gcd(a , b);
}

int main(){
    std::cout << "Lowes common multiple for 2 values" << std::endl;
    std::cout << "Input a: ";
    unsigned int a;
    std::cin >> a;
    std::cout << "Input b: ";
    unsigned int b;
    std::cin >> b;

    unsigned int result = lcm(a, b);
    std::cout << "lcm(a, b) = " << result << std::endl;

} 